app/models/team_member.rb

```diff
@@ -8,4 +8,28 @@ class TeamMember < ApplicationRecord
   def name
     first_name
   end
+
+  def job_title
+    if discipline == "Development"
```

Comment:
Another way to do this would be the [`case` statement](https://www.geeksforgeeks.org/ruby-case-statement/), which might be cleaner in this instance.